core/embed/rust/src/ui/model_tt/component/scroll.rs

```diff
@@ -21,7 +21,8 @@ impl ScrollBar {
     /// Center to center.
     const DOT_INTERVAL: i16 = 18;
 
-    fn new(layout: LinearPlacement) -> Self {
+    pub fn new(axis: Axis) -> Self {
```

Comment:
This `new` seems to not be used outside - has the `pub` some other idea? Seems little clearer to hide it when not used

Reply:
It's used in [simple_page.rs](https://github.com/trezor/trezor-firmware/blob/fc4f8bfa70aaeaba943306318c592df95a05ead3/core/embed/rust/src/ui/model_tt/component/simple_page.rs#L32).